net/libreswan/Makefile

```diff
@@ -64,7 +66,6 @@ endef
 TARGET_CFLAGS+= -Wno-error=format-nonliteral
```

Comment:
No idea if this uses autotools or what. Does this support passing

```
--disable-warnings
```

to CONFIGURE_ARGS? I'd rather that be used.

Reply:
it does not use autotools.

Reply:
it was the best I knew at that moment, I didn't test it other ways

Reply:
same feeling here.

Reply:
I made a compile on x86 and it worked with without TARGET_CFLAGS+= -Wno-error=format-nonliteral, maybe the new version fixed the error, I think it was in the kernel module, don't remember exactly

Reply:
@antonyantony ping on this

Reply:
> I made a compile on x86 and it worked with without TARGET_CFLAGS+= -Wno-error=format-nonliteral, maybe the new version fixed the error, I think it was in the kernel module, don't remember exactly

thanks for the suggestion. Let me try and get back  in  few hours. May be it is not necessary anymore.  we should compile for 32bit  to be sure.

Reply:
I've run tested it on an atom 32 bit target